### DISCUSSION TITLE
Apple Voice Memos transcription

### DISCUSSION THREAD

**Discussion by michaelgriscom** [openai/whisper#668] [Show and tell]
Just hacked together a script to transcribe Apple Voice Memos:
https://github.com/michaelgriscom/voice-memo-transcription

My use case is recording notes on my Apple Watch when I'm listening to audiobooks on the go, which I then export to PKM software.

I'm a big fan of Whisper, huge thanks to OpenAI!

**Comment by roboslone** [comment]
I've written a simple service to run this in background: https://github.com/roboslone/landline
Transcription code is taken from linked script :)

**Comment by Jawschamp** [comment]
> I'm a big fan of Whisper, huge thanks to OpenAI!

xAi's will be better, trust
